cypress/e2e/3-learn/home.cy.js

```diff
@@ -0,0 +1,18 @@
+/// <reference types="cypress" />
+
+describe('my awesome website', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000')
+  })
+
+  it('turns page to dark mode', () => {
+    cy.get('.Theme-Toggle').check()
+  })
+  it('turns page to light mode', () => {
+    cy.get('.Theme-Toggle').uncheck()
+  })
+
+  it('sorts numbers', () => {
```

Comment:
This here belongs in another test file. Keep each file specific to a single User Story Description where possible.